SPECS/kernel-alt.spec

```diff
@@ -1062,6 +1095,7 @@ fi
 * Fri Jun 21 2024 Thierry Escande <thierry.escande@vates.tech> - 4.19.316+1-1
 - Import kernel source v4.19.316
 - Rebase and rework needed patches from main kernel repo
```

Comment:
Please indicate the exact release of the main kernel we synced with. This will help know when we last synchronised. Especially now that the changelog entry date is *before* the date when the main kernel was last updated.